src/recval/evaluator.py

```diff
@@ -3,28 +3,30 @@
 import logging
 from dataclasses import dataclass, field
 
-import numpy as np
+import numpy
 import numpy.typing as npt
-import pandas as pd
-from pytictoc import TicToc
+import pandas
 
 from recval.constants import DEFAULT_ITEM_COL, DEFAULT_USER_COL
 from recval.metrics.metric_interface import MetricsEnum
 from recval.metrics.metrics_utils import get_hit_rank
 from recval.utils import get_topk
 
+from .decorators import timeit
```

Comment:
How do you choose when use a relative import with respect to the full one?

Reply:
Di solito fino a un singolo punto va bene. finche' stai all'interno del modulo puoi usare import relativi